packages/frontend/src/view/components/safe/SafeMultisigTransaction.tsx

```diff
@@ -52,33 +53,48 @@ export function SafeMultisigTransaction({
   return (
     <div
       className={cx(
-        'col-span-5 grid min-w-[800px] grid-cols-multisig border-b border-gray-700 py-3 text-xs',
+        'col-span-full grid-cols-multisig-small border-b border-gray-700 py-3 text-xs last:rounded-b last:border-none md:min-w-[800px] md:grid-cols-multisig  ',
         isExpanded ? 'rounded border-none bg-gray-750' : 'bg-gray-800',
       )}
     >
       <div
-        className="col-span-5 grid min-w-[800px] cursor-pointer grid-cols-multisig"
+        className="grid cursor-pointer grid-cols-multisig-small md:min-w-[800px] md:grid-cols-multisig"
         onClick={() => setIsExpanded(!isExpanded)}
       >
-        <div className="flex items-center px-6">
+        <div className="flex items-center px-4 md:px-6">
           {getTimeDifference(new Date(transaction.submissionDate))}
         </div>
-        <div className="flex items-center font-bold">{method}</div>
+        <div className="hidden items-center font-bold md:flex">{method}</div>
         <div className={cx('flex items-center', statusToTextColor(txStatus))}>
           {stringConfirmations}
         </div>
         <StatusBadge status={txStatus} />
-        <div>
-          <button className="brightness-100 filter transition-all duration-300 hover:brightness-[120%]">
-            {isExpanded ? <SolidMinusIcon /> : <SolidPlusIcon />}
-          </button>
-        </div>
+        <button className="brightness-100 filter transition-all duration-300 hover:brightness-[120%]">
+          {isExpanded ? <SolidMinusIcon /> : <SolidPlusIcon />}
+        </button>
       </div>
 
       {isExpanded && (
-        <div className="col-span-5 mt-3 cursor-auto">
+        <div className="col-span-full mt-3 cursor-auto">
           <TransactionProperty param="Submission date" value={submissionDate} />
           <TransactionProperty param="Execution date" value={executionDate} />
+          {transaction.blockNumber && (
+            <TransactionProperty
+              param="Block number"
+              value={
+                <BlockNumber blockNumber={transaction.blockNumber} noStyle />
+              }
+            />
+          )}
+          <TransactionProperty
+            param="Transaction hash"
+            value={
+              <TransactionHash
+                transactionHash={transaction.transactionHash}
+                noStyle
+              />
+            }
+          />
```

Comment:
If the transaction has not been executed, the transaction hash will be null thus crashing frontend since we have no error boundaries

Reply:
To reproduce please choose any discarded transaction of your liking and try to expand it

Reply:
Yet again safe api kit typings are wrong, I'll submit a fix soonish ;p

Reply:
[77c77c4](https://github.com/l2beat/lz-monitoring/pull/102/commits/77c77c40cf49ac4aed931dc64141c79906db0aa4)